index.html

```diff
@@ -164,9 +180,9 @@ <h2>LET’S CONNECT</h2>
           <div class="img5"><img src="./image/Ellipse5.png" alt="img" ></div>
 
         
-               <div class="contact-form">
-                 
-                <h1 class="intro">Contact Me</h1>
+          <div class="contact-form">
```

Comment:
- [ ] The contact section has no background image, please add the correct one, so your design can be like the provided one
- [ ] The elements of the form should be centered on the page including the ``` Get in touch``` button, please fix this

- Your design: 
![Screenshot_20220507_120257](https://user-images.githubusercontent.com/25227022/167253511-baf9a22f-91ef-4fba-82ca-529bcc30fc32.png)